bin/setup

```diff
@@ -17,13 +17,13 @@ FileUtils.chdir APP_ROOT do
   system! "gem install bundler --conservative"
   system("bundle check") || system!("bundle install")
 
-  # Install JavaScript dependencies
-  # system("bin/yarn")
+  puts '== Installing JavaScript dependencies =='
+  system('bin/yarn')
 
-  # puts "\n== Copying sample files =="
-  # unless File.exist?("config/database.yml")
-  #   FileUtils.cp "config/database.yml.sample", "config/database.yml"
-  # end
+  puts "\n== Copying sample files =="
+  unless File.exist?("config/database.yml")
+    FileUtils.cp "config/database.yml.sample", "config/database.yml"
+  end
```

Comment:
database.yml is tracked in git already. It would require someone to delete that file for this to become useful.

I would add `.env.example` => `.env` here though.